pkg/generators/openapi_test.go

```diff
@@ -378,6 +378,7 @@ func TestCustomDefs(t *testing.T) {
 	callErr, funcErr, assert, callBuffer, funcBuffer := testOpenAPITypeWriter(t, `
 package foo
 
+// Blah is a custom type
 type Blah struct {
```

Comment:
have you tried to add a `json:"-"` tag? Does kube-openapi cope with that?

Reply:
not sure what you're asking... this is testing a type that does its own custom marshaling, and so overrides the type/format presented in openapi. this PR is allowing that type to still include documentation.

Reply:
I meant if you create a field `Blah Blah 'json:"-"'`. Will kube-openapi ignore that json tag? It's a general question, not so much about this PR.

Reply:
ah, I don't know, I didn't try